pyproject.toml

```diff
@@ -143,6 +143,8 @@ max-statements = 60
 disable = [
   # Disabled on purpose:
   "line-too-long", # covered by black
+  "no-else-raise", # conflicts with ruff TRY
+  "no-else-return", # conflicts with ruff TRY
```

Comment:
I don't think these are actually conflicts... you _can_ move them to an `else` block to get that section out of the `try` block, but all of these are trailing `else` blocks that can be dropped entirely and keep the same outcome.